scraper.py

```diff
@@ -0,0 +1,46 @@
+__author__ = 'shrenilpatel'
+
+import json
+import urllib2
+
+#FOOD SERVICE
+getUrl = "https://api.uwaterloo.ca/v2/foodservices/locations.json?key=a19ad7d4ee67feebd6b7caa461251302"
+#postUrl = 'https://geotaganything.herokuapp.com/tags' #ACTUAL
+postUrl = 'http://localhost:5000/tags' #LOCAL
```

Comment:
No this was just a quick and dirty example to get from UWAPI and post to Heroku. API has changed so this all doesnt really work unless your local is running the version before API changes. It's not very useful ATM but its a start for anyone interested in the task. The idea was you'd change the getUrl and get method and just run the script once. Though there is benefit from keeping them all